apps/whale-api/src/module.api/poolpair.fees.service.ts

```diff
@@ -0,0 +1,69 @@
+import { Inject, Injectable } from '@nestjs/common'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import BigNumber from 'bignumber.js'
+import { SemaphoreCache } from '@defichain-apps/libs/caches'
+import { EstimatedDexFee, SwapPathWithDirection } from '@defichain/whale-api-client/dist/api/poolpairs'
+import { PoolSwapAggregatedMapper } from '../module.model/pool.swap.aggregated'
+import { TransactionVoutMapper } from '../module.model/transaction.vout'
+import { NetworkName } from '@defichain/jellyfish-network'
+import { DeFiDCache } from './cache/defid.cache'
+
+@Injectable()
+export class PoolPairFeesService {
+  constructor (
+    @Inject('NETWORK') protected readonly network: NetworkName,
+    protected readonly rpcClient: JsonRpcClient,
+    protected readonly deFiDCache: DeFiDCache,
+    protected readonly cache: SemaphoreCache,
+    protected readonly poolSwapAggregatedMapper: PoolSwapAggregatedMapper,
+    protected readonly voutMapper: TransactionVoutMapper
+  ) {
+  }
+
+  public async getDexFees (tokenPathWithDirection: SwapPathWithDirection): Promise<EstimatedDexFee[] | []> {
+    const dexFees: EstimatedDexFee[] = []
+    const { tokenFrom, tokenTo, poolPairId } = tokenPathWithDirection
+
+    const attrs = (await this.rpcClient.masternode.getGov('ATTRIBUTES')).ATTRIBUTES
```

Comment:
Possible to rely on deFiDCache for this? Since the attributes seldom change, I think relying on cache will be more performant.

Reply:
Possible and updated! But, needed to combine -> split id to comply with `this.get`